interface/billing/billing_report.php

```diff
@@ -705,14 +733,14 @@ class='link_submit'  ><?php echo '['.xlt('End Of Day Report').']' ?></a>
     $mmo_num_charges = 0;
 
     foreach ($ret as $iter) {
-        // We include encounters here that have never been billed.  However
+        // We include encounters here that have never been billed. However
         // if it had no selected billing items but does have non-selected
         // billing items, then it is not of interest.
-        if (!$iter['id']) {
-            $res = sqlQuery("SELECT count(*) AS count FROM billing WHERE " .
-            "encounter = ? AND " .
-            "pid=? AND " .
-            "activity = 1", array($iter['enc_encounter'],$iter['enc_pid']));
+        if (! $iter['id']) {
+            $res = sqlQuery("SELECT count(*) AS count FROM billing WHERE " . "encounter = ? AND " . "pid=? AND " . "activity = 1", array(
```

Comment:
there's appears to be some weirdness (ie. auto changes) where lines, such as above are getting combined. there are some more obvious examples of this below in this script.

Reply:
This is definitely an odd thing that has happened. It seems like there was auto changed done here and below that concatenated these queries which has caused them from going from readable to no longer readable. Rec. looking at the script with the ignore whitespacing via this link to see what I mean:
https://github.com/openemr/openemr/pull/1014/files?w=1